stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -601,17 +601,23 @@ def mksquashfs_metal(paths, workdir, img_metal, loop_client):
             print(f"Kernel binary linked: {tmp_squashfs_dir}/boot/{kernel_binary_basename}")
             print(f"Kernel HMAC linked: {tmp_squashfs_dir}/boot/{kernel_hmac_basename}")
             # Generate root squashfs
-            print(f'Compressing squashfs with {squashfs_compression}')
+            print(f'Creating {fstype} with {fsoptions}')
 
             # Note the filename must be exactly "root.squashfs" because the 20live
```

Comment:
```suggestion
            # Note the filename must be exactly "root.squashfs" or "root.erofs" because the 20live
```

might have to rework spacing on this and subsequent lines.

Reply:
will resolve in a followup PR